subgraphs/bar/bar.fuse.yaml

```diff
@@ -0,0 +1,31 @@
+specVersion: 0.0.3
+description: Bar
+repository: https://github.com/traderjoe-xyz/joe-subgraph
+schema:
+  file: ./bar.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: JoeBar
+    network: fuse
+    source:
+      address: '0x97a6e78c9208c21afaDa67e7E61d7ad27688eFd1'
+      abi: Bar
+      startBlock: 15542382
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - Bar
+        - User
+      abis:
+        - name: Bar
+          file: ../../node_modules/@traderjoe-xyz/core/abi/JoeBar.json
```

Comment:
you need to use the VoltBar, VoltToken, VoltPair abis

Reply:
done, not sure if I also need to change ABI names so left that part out. Got the VoltBar abi from https://explorer.fuse.io/address/0x97a6e78c9208c21afaDa67e7E61d7ad27688eFd1/contracts and the other two from [@voltage-finance/core](https://www.npmjs.com/package/@voltage-finance/core)